WebService/Properties/launchSettings.json

```diff
@@ -2,23 +2,22 @@
   "profiles": {
     "WebService": {
       "commandName": "Project",
-      "launchBrowser": false,
       "launchUrl": "http://localhost:9003/v1/status",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development",
-        "PCS_IOTHUB_CONNSTRING": "your Azure IoT Hub connection string",
+        "PCS_SUBSCRIPTION_ID": "",
+        "PCS_RESOURCE_GROUP": "",
+        "PCS_IOHUB_NAME": "",
         "PCS_STORAGEADAPTER_WEBSERVICE_URL": "http://localhost:9022/v1",
-        "PCS_LOG_LEVEL": "Debug",
+        "PCS_IOTHUB_CONNSTRING": "HostName=isaactest1f9136.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=dbVWLmLfmBJyeT3h0mKV+/eoSBLE6TqBr2c7lgbSFUs=",
```

Comment:
remove this and invalidate the credentials ASAP, make sure these credentials are not valid